vstsBuild/vstsBuild.psm1

```diff
@@ -80,31 +80,33 @@ function Invoke-VstsPublishBuildArtifact
         # Only publish files once
         if($script:publishedFiles -inotcontains $fileName)
         {
-            $leafFileName = $(Split-path -Path $FileName -Leaf)
+            $leafFileName = $(Split-path -Path $fileName -Leaf)
 
-            $extension = [System.io.path]::GetExtension($fileName)
-            if($extension -ieq '.zip')
+            $extension = [System.IO.Path]::GetExtension($leafFileName)
+            $nameWithoutExtension = [System.IO.Path]::GetFileNameWithoutExtension($leafFileName)
+            # Only expand the symbol '.zip' package
+            if($extension -ieq '.zip' -and $nameWithoutExtension.Contains("symbols"))
             {
-                $unzipPath = (Join-Path $destinationPath -ChildPath $leafFileName)
+                $unzipPath = (Join-Path $destinationPath -ChildPath $nameWithoutExtension)
                 if($Variable)
                 {
                     Write-VstsInformation -message "Setting VSTS variable '$Variable' to '$unzipPath'"
                     # Sets a VSTS variable for use in future build steps.
                     Write-Host "##vso[task.setvariable variable=$Variable]$unzipPath"
-                    # Set a variable in the current process.  PowerShell will not pickup the variable until the process is restarted otherwise.
+                    # Set a variable in the current process. PowerShell will not pickup the variable until the process is restarted otherwise.
                     Set-Item env:\$Variable -Value $unzipPath
                 }
                 Expand-Archive -Path $fileName -DestinationPath $unzipPath
             }
 
-            Write-Host "##vso[artifact.upload containerfolder=results;artifactname=$leafFileName]$FileName"
+            Write-Host "##vso[artifact.upload containerfolder=results;artifactname=$leafFileName]$fileName"
```

Comment:
can we update the container folder to be `$Bucket`?

Reply:
Updated. Use `$Bucket` instead.